lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -18,6 +18,28 @@
       &.is-hidden {
         display: none;
       }
+
+      &.is-disabled {
+        a {
+          pointer-events: none;
+
+          .action-icon {
+            display: none;
+          }
+
+          .action-label {
+            padding-right: 0px;
```

Comment:
Nit: I wonder if it would be cleaner to have the padding on the icon, because then it would naturally go away when the icon is hidden.